src/tests/testing/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+set(go_tests
+    # Add here your tests files
+    GOTestOrganModel.cpp
+    GOTestSwitch.cpp
+    GOTestWindchest.cpp
```

```suggestion
    model/GOTestOrganModel.cpp
    model/GOTestSwitch.cpp
    model/GOTestWindchest.cpp
```

The testing files should be distributed across subdirectories as like the source files are.

---

> I think the do-tests.sh script should do all these commands:

Usually, it's better to not run the coverage if tests failed (that's the case in the github workflow), that's why I splitted the commands. That's also the case when developing (usually I don't care about coverage until push on git).